src/components/ModalBackupWords.js

```diff
@@ -111,14 +95,15 @@ class ModalBackupWords extends React.Component {
         const { storage } = getGlobalWallet();
         const accessData = await storage.getAccessData();
         const words = hathorLib.cryptoUtils.decryptData(accessData.words, password);
-        this.props.updateWords(words);
+        this.state.words = words;
```

Comment:
[not blocking] Why not use the `this.setState` here? I feel the it would be better.

Reply:
You actually caught a bug waiting to happen: I was updating the state incorrectly and it could not trigger the re-rendering of the component.

Fixed on https://github.com/HathorNetwork/hathor-wallet/pull/608/commits/afb58ccc5cc1ed5358377df85ff0be2cb267e2da.